CMakeLists.txt

```diff
@@ -36,6 +36,15 @@ option(fsanitize_address "enable /fsanitize=address compiler switch (default is
 option(build_csharp_projects "Build csharp projects? (default is ON)" ON)
 option(build_sarif_results_checker "Build Sarif results checker? (default is ON)" ON)
 
+if ("$ENV{BUILD_BINARIESDIRECTORY}" STREQUAL "")
+    message(WARNING "BUILD_BINARIESDIRECTORY not defined!")
```

Comment:
>BUILD_BINARIESDIRECTORY not defined [](http://example.com/codeflow?start=21&length=35)

for the people who have no idea what this env variable is expected to contain, it would be really nice to provide an explanation of what it should contain (in this case, it is the name of a folder).

Reply:
Added a comment explaining it